bin/react-docgen.js

```diff
@@ -9,55 +9,45 @@
  *
  */
 
-var argv = require('nomnom')
-  .script('react-docgen')
-  .help(
-    'Extract meta information from React components.\n' +
-    'If a directory is passed, it is recursively traversed.'
-  )
-  .options({
-    path: {
-      position: 0,
-      help: 'A component file or directory. If no path is provided it reads from stdin.',
-      metavar: 'PATH',
-      list: true,
-    },
-    out: {
-      abbr: 'o',
-      help: 'store extracted information in FILE',
-      metavar: 'FILE',
-    },
-    pretty: {
-      help: 'pretty print JSON',
-      flag: true,
-    },
-    extension: {
-      abbr: 'x',
-      help: 'File extensions to consider. Repeat to define multiple extensions. Default:',
-      list: true,
-      default: ['js', 'jsx'],
-    },
-    excludePatterns: {
-      abbr: 'e',
-      full: 'exclude',
-      help: 'Filename pattern to exclude. Default:',
-      list: true,
-      default: [],
-    },
-    ignoreDir: {
-      abbr: 'i',
-      full: 'ignore',
-      help: 'Folders to ignore. Default:',
-      list: true,
-      default: ['node_modules', '__tests__', '__mocks__'],
-    },
-    resolver: {
-      help: 'Resolver name (findAllComponentDefinitions, findExportedComponentDefinition) or path to a module that exports a resolver.',
-      metavar: 'RESOLVER',
-      default: 'findExportedComponentDefinition',
-    },
-  })
-  .parse();
+var argv = require('commander');
+
+function collect(val, memo) {
+  memo.push(val);
+  return memo;
+}
+
+argv
+    .option(
+        '--path <PATH>',
```

Comment:
path was not an option but an argument
from the help with nomnom:
```
Usage: react-docgen [path]... [options]
```

So this option would be new, and is also unused.
seems that `[FILES]` is the equivalent to `[path]`